terraform/cloudfront.tf

```
@@ -0,0 +1,100 @@
+
+locals {
+  s3_static_origin_id = "ashley-static-origin"
```

Statics are not hosted on S3

```suggestion
  static_origin_id = "ashley-static-origin"
```